.circleci/config.yml

```
@@ -73,11 +73,11 @@ aliases:
     # Nota bene: you'll need to define the following secrets environment vars
     # in CircleCI interface:
     #
-    #   - DOCKER_USER
-    #   - DOCKER_PASS
+    #   - DOCKER_HUB_USER
```

Why not ?
```suggestion
    #   - DOCKER_HUB_USERNAME
```

the `-u` is the shortcut for `--username`